app/src/components/Section.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <section :id="setidentifier + '-section'" class="app-section">
+    <div class="app-section-content">
+      <slot></slot>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  name: 'PageSection',
+  props: ['id'],

```

Comment:
type

Reply:
id should be required or defaulted, otherwise line 15 might throws 'cannot read property toLowerCase of undefined'